client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-site-metrics.tsx

```diff
@@ -1,39 +1,48 @@
 import { useIsEnglishLocale } from '@automattic/i18n-utils';
 import { getQueryArg } from '@wordpress/url';
 import { useUrlBasicMetricsQuery } from 'calypso/data/site-profiler/use-url-basic-metrics-query';
-import { upgradePlanSiteMetricsLcpThreshold } from '../constants';
+import {
+	upgradePlanSiteMetricsLcpThreshold,
+	upgradePlanSiteMetricsFidThreshold,
+} from '../constants';
+
+const calcPercentageDifferenceFromThreshold = ( value: number | undefined, threshold: number ) => {
```

Comment:
Calc being discussed in https://github.com/Automattic/wp-calypso/pull/90457#discussion_r1600725701